Unit-01/02-flask-routing/app.py

```diff
@@ -0,0 +1,46 @@
+from flask import Flask, render_template, request
+
+app = Flask(__name__)
+
+@app.route("/")
+def welcome():
+  return "Welcome to my app"
+
+# @app.route("/add/<int:num1>/<int:num2>")
+# def add(num1, num2):
+#   return f"The sum of {num1} + {num2} is {num1 + num2}" 
+
+# @app.route("/multiply/<int:num1>/<int:num2>")
+# def multiply(num1, num2):
+#   return f"The product of {num1} * {num2} is {num1 * num2}"
+
+# @app.route("/divide/<int:num1>/<int:num2>")
+# def divide(num1, num2):
+#   return f"The divition of {num1} / {num2} is {num1 / num2}"
+
+# @app.route("/substruct/<int:num1>/<int:num2>")
+# def substruct(num1, num2):
+#   return f"The substruction of {num1} - {num2} is {num1 - num2}" 
+
+@app.route("/calculate")
+def calc():
+	return render_template("calc.html")
+
+@app.route("/math")
+def math():
```

Comment:
this section should be structured like this next time:

```python
@app.route("/math/<operation>/<int:num1>/<int:num2>")
```

Since we want to do the operation using the route instead of a query string